frontend/src/routeLoaders.ts

```diff
@@ -207,6 +207,7 @@ export const getClassMgmtData: LoaderFunction = async ({
     const { class_id } = params;
     let cls: Class | undefined;
     let attendanceRate: number | undefined;
+    let missingAttendance: number | undefined;
```

Comment:
@calisio I couldn't comment on the line I wanted to, but what do you think about leaving the value as 0 for attendance rate and missing attendance if the class is still scheduled? It cuts down on 2 unnecessary requests. Worth it?